packages/navigation/tests/Navigation.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
```

Comment:
@branclar Should this file be named `Navigation.test.tsx` not `Navigation.stories.tsx`? A proposal for fix in https://github.com/project44/manifest/pull/216

Reply:
yep! Will update